spec/integration/auth_login_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'swagger_helper'
```

Comment:
- [ ]  If you include in your readme this command for your tests:

```bash
rake rswag:specs:swaggerize
```

![image](https://user-images.githubusercontent.com/57421823/198724783-f2ef45b9-5abf-49c1-82b1-5f6db3ef657d.png)


Absolutely all your tests will be run including the swagger ones.  That way, your users get a clear picture that all are passing.